pkg/cli/globalflags/globalflags.go

```diff
@@ -8,6 +8,8 @@ import (
 type GlobalOptions struct {
```

Comment:
Nothing wrong here, but I feel that name does not align with `GlobalOptions` anymore since it contains `url`/`proxy`/`manager` & `service` . May be `GlobalLoginOptions` or `LoginOptions`

Reply:
We might add a few more flags on feature requests in the future, having a single data store with a more generic name (`GlobalOptions`) seems like a better option than confining to a specific use-case.